documentation/modules/ROOT/pages/connectors/db2.adoc

```diff
@@ -2002,18 +2002,34 @@ See {link-prefix}:{link-db2-connector}#db2-data-types[Db2 data types] for the li
 
 |[[db2-property-message-key-columns]]<<db2-property-message-key-columns, `+message.key.columns+`>>
 |_empty string_
-|A semicolon separated list of tables with regular expressions that match table column names. The connector maps values in matching columns to key fields in change event records that it sends to Kafka topics. This is useful when a table does not have a primary key, or when you want to order change event records in a Kafka topic according to a field that is not a primary key. +
+|A list of expressions that specify the columns that the connector uses to form custom message keys for change event records that it publishes to the Kafka topics for specified tables.
+
+By default, {prodname} uses the primary key column of a table as the message key for records that it emits.
+In place of the default, or to specify a key for tables that lack a primary key, you can configure custom message keys based on one or more columns. +
+ +
+To establish a custom message key for a table, list the table, followed by the columns to use as the message key.
+Each list entry takes the following format: +
+ +
+`_<fully-qualified_tableName>_:_<keyColumn>_,_<keyColumn>_` +
+ +
+To base a table key on multiple column names, insert commas between the column names.
+
+Each fully-qualified table name is a regular expression in the following format: +
  +
-Separate entries with semicolons. Insert a colon between the fully-qualified table name and its regular expression. The format is: +
+`_<schemaName>_._<tableName>_` +
  +
-_schema-name_._table-name_:_regexp_;... +
+The property can include entries for multiple tables.
+Use a semicolon to separate table entries in the list. +
  +
-For example, +
+The following example sets the message key for the tables `inventory.customers` and `purchase.orders`: +
```

Comment:
"and `purchase.orders`" -> "and tables named `purchaseorders` in any schema"

At least that's what your example shows with the regex in the the schema part.